core/lib/Cardano/Address/Style/Shelley.hs

```diff
@@ -147,12 +155,20 @@ import GHC.Generics
     ( Generic )
 
 import qualified Cardano.Address.Derivation as Internal
+import qualified Cardano.Address.Script as Script
 import qualified Cardano.Address.Style.Byron as Byron
 import qualified Cardano.Address.Style.Icarus as Icarus
+import Cardano.Codec.Bech32
+    ( FromBech32 (..), ToBech32, bech32, bech32With, fromBech32With )
+import qualified Cardano.Codec.Bech32.Prefixes as CIP5
 import qualified Data.Aeson as Json
 import qualified Data.ByteArray as BA
 import qualified Data.ByteString as BS
 import qualified Data.ByteString.Lazy as BL
+import Data.Coerce
+    ( coerce )
+import Data.Either.Extra
+    ( maybeToEither )
```

Comment:
On grouping imports, have a look at: https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas

Do not interleave qualified with explicit imports :+1:

Reply:
I was hoping I could just not look at imports and stylish-haskell would take care of it.

Reply:
> I was hoping I could just not look at imports and stylish-haskell would take care of it.

It will ... **_mostly_**. 😅 

But AFAIK, qualified imports must be placed at the end manually.

It's an annoyance, I agree.

Reply:
stylish-haskell will do most of it -- it you're using the version which works (see my other comment). But you need to group the imports yourself.